### PR TITLE
Swan new bfs

### PR DESCRIPTION
**Pull request by bee-san**

### CODE REVIEW

File: src/searchers/bfs.rs [hunk 1]
```diff
@@ -38,8 +61,13 @@ mod tests {
...
-        let result = bfs("Q0FOQVJZOiBoZWxsbw==");
+        // this will work after english checker can identify "CANARY: hello"
+        // let result = bfs("Q0FOQVJZOiBoZWxsbw==");
+        // assert!(result.is_some());
+        // assert!(result.unwrap() == "CANARY: hello");
+
+        let result = bfs("b2xsZWg=");
         assert!(result.is_some());
-        assert!(result.unwrap() == "CANARY: hello")
+        assert!(result.unwrap() == "hello");
```
Comment (bee-san) [inline src/searchers/bfs.rs:71]: Can I ask why this change?

Comment (swanandx) [reply src/searchers/bfs.rs:71]: There is a comment above it ☺️ This is changed due to English checker couldn't identify `CANARY: hello`. Once we get the new and better English checker, we can just use the old test ( it's just commented out )!
